StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentOption+Images.swift

```diff
@@ -84,6 +84,12 @@ extension STPPaymentMethod {
             return PaymentSheetImageLibrary.bankIcon(
                 for: PaymentSheetImageLibrary.bankIconCode(for: usBankAccount?.bankName)
             )
+        case .link:
+            if let linkPaymentDetails {
+                return STPImageLibrary.cardBrandImage(for: linkPaymentDetails.brand)
```

Comment:
Same question here? I find it a bit confusing that it's just called "linkPaymentDetails" but is always a card?

Reply:
Answered above.